src/main/java/baritone/process/MineProcess.java

```diff
@@ -74,6 +78,37 @@ public boolean isActive() {
 
     @Override
     public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
+        if (Baritone.settings().checkInventory.value) {
```

Comment:
Is this mostly the same as the one in FarmProcess? Can it be moved out to a utility?

Reply:
i can try to move this out to the processhelper, but i don't know yet if this will work, because there are small differences

Reply:
nvm this was kinda easy